src/presets/cloudflare.ts

```diff
@@ -8,7 +8,7 @@ export const cloudflare = defineNitroPreset({
   entry: '#internal/nitro/entries/cloudflare',
   commands: {
     preview: 'npx miniflare ./server/index.mjs --site ./public',
-    deploy: 'cd ./server && npx wrangler publish'
+    deploy: 'npx @cloudflare/wrangler publish'
```

Comment:
do we not need to cd with `@cloudflare/wrangler`?

Reply:
Actually not it was not needed since the `wrangler.toml` actually gives output to read from